feconf.py

```diff
@@ -951,11 +952,19 @@ def get_empty_ratings():
 HANDLER_TYPE_HTML = 'html'
 HANDLER_TYPE_JSON = 'json'
 
-# Following are the constants for the roles.
+# Following are the constants for the roleIds. RoleIds are human
+# readable roles with underscores instead of spaces.
+# To check the role updation from /admin#roles, set ADMIN_SHOW_UPDATE_ROLE
+# constant in Admin.js to true.
 ROLE_GUEST = 'GUEST'
 ROLE_BANNED_USER = 'BANNED_USER'
 ROLE_EXPLORATION_EDITOR = 'EXPLORATION_EDITOR'
 ROLE_COLLECTION_EDITOR = 'COLLECTION_EDITOR'
 ROLE_MODERATOR = 'MODERATOR'
 ROLE_ADMIN = 'ADMIN'
 ROLE_SUPER_ADMIN = 'SUPER_ADMIN'
+
+# Intent of the User making query to role structure via admin interface. Used
+# to store audit data regarding queries to roleIds.
+VIEW_ROLE = 'View user roles.'
```

Comment:
ROLE_ACTION_VIEW_ROLE, ROLE_ACTION_UPDATE_ROLE?

Also just 'view' and 'update' should suffice. These are backend entries that are not viewable by users.